README.md

```diff
@@ -32,15 +32,19 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. If you were to describe semantic HTML to the next cohort of students, what would you say?
+1. If you were to describe semantic HTML to the next cohort of students, what would you say? 
+I would say it makes it readable for other developers and organizes your html so its not a jumbled mess.
```

Comment:
Don't forget it's also used to help with search engine rankings, as the search engine can use the semantic tags to help analyze and index your site for searches